pkg/sanity/controller.go

```diff
@@ -1254,21 +1254,6 @@ var _ = DescribeSanity("ListSnapshots [Controller Server]", func(sc *SanityConte
 		Expect(snapshots.GetEntries()).To(BeEmpty())
 	})
 
-	It("should fail when an invalid starting_token is passed", func() {
-		vols, err := c.ListSnapshots(
-			context.Background(),
-			&csi.ListSnapshotsRequest{
-				StartingToken: "invalid-token",
-			},
-		)
-		Expect(err).To(HaveOccurred())
-		Expect(vols).To(BeNil())
-
-		serverError, ok := status.FromError(err)
-		Expect(ok).To(BeTrue())
-		Expect(serverError.Code()).To(Equal(codes.Aborted))
-	})
-
```

Comment:
I think we still need some kind of invalid token test?  I thought you want to remove the next test "should fail when the starting_token is greater than total number of snapshots".

Reply:
Just talked to Jing.  We don't know what kind of token a driver will use so we don't really know what is invalid.  So this test can be removed.